inference-engine/tests/functional/shared_test_classes/src/single_layer/region_yolo.cpp

```diff
@@ -3,11 +3,13 @@
 //
 
 #include "shared_test_classes/single_layer/region_yolo.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
 
 namespace LayerTestsDefinitions {
+using namespace ov::test;
 
 std::string RegionYoloLayerTest::getTestCaseName(const testing::TestParamInfo<regionYoloParamsTuple> &obj) {
-    ngraph::Shape inputShape;
+    ov::test::InputShape inputShape;
```

Comment:
First you use `using namespace ov::test` but then you use `ov::test::InputShape`, it's strange...

Reply:
type full name has been used before `using namespace ov::test` was added. Fixed, thanks